test/test_helper.rb

```diff
@@ -1,8 +1,8 @@
-require "minitest/autorun"
-require "flexmock/test_unit"
-require "anyplayer"
+require 'minitest/autorun'
+require 'flexmock/test_unit'
+require 'anyplayer'
 
-require "anyplayer/players/noplayer"
-Anyplayer::PLAYERS.unshift "noplayer"
+require 'anyplayer/players/noplayer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.